_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,147 @@
+  It is a great example of how a team was able to work in the open to benefit the government as a whole but also the society itself.
+- [GC Service & Digital Target Enterprise Architecture](https://wiki.gccollab.ca/index.php?title=GC_Service_%26_Digital_Target_Enterprise_Architecture&mobileaction=toggle_view_desktop){:target="_target"}
+- [1988 National Bureau of Standards' Management Guide to
+Software Reuse](https://www.govinfo.gov/content/pkg/GOVPUB-C13-dca04f7b7fc21321e39de88b35b2f1d3/pdf/GOVPUB-C13-dca04f7b7fc21321e39de88b35b2f1d3.pdf) (PDF)
+
+Needless to say that public administrations in general are ripe for reusing solutions and ideas to increase the quality of their services to citizens.
+
+### Manage secrets
+
+In software development, certain parts of the code may be a corporate secret, such as a private key to a resource, passwords, etc.
+
+These must never be included in the public source code itself but rather be managed separately.
+
+Since, software source code is nothing more than structured text that can be understood by a computer, the same concept of pulling out secrets parts may be taken with other types of documents.
+
+This means that when a small part of your work must not be shared publicly, you can exclude it and store it in a secure location instead of locking down the entire document.
+
+This way, the material that is unclassified can still be made available publicly and may be collectively worked on while the parts that are to be secured can be edited separately.
+
+By default, you should strive to keep as many of your working documents in the open section of your workspace.
+But when needed, you can save sensitive information in the private folders and keep working in the open for the rest.
+
+### Manage leaks
+
+In software development, we expect that there will be a security issue at some point, while we continue to strive to minimize that eventuality.
+
+By putting in place a remediation process, we ensure that even if there was to be a leak, appropriate mesures are in place to address it as quickly as possible.
+And we make sure that everyone knows how to run through the process, if it's not fully automated.
+
+### Open up at inception
+
+Part of working in the open is accepting that we don't know everything and that others may be able to jump in and help at any time.
+
+When one goes into draft mode, it's easy to think that we can do better before getting feedback, so the initial reaction most people have is to wait until "it's good enough".
+
+If you are worried to be slowed down by feedback on a document that is barely an outline, you probably can comfort yourself with the thought that, in practice, chances are that contributions would not come before your product or document is actually mature enough!
+
+Plus, waiting too long may put you in the position where you have not set your mind into an open mode, continuously expecting someone to read it.
```

Comment:
I wonder if this could be more like....'Additionally, as you get closer to a final version of a document, you may feel less open to accepting comments and feedback on your work".   Feel like there must be some research to back this up. Will look....

Reply:
Ah yes, good point!